src/Sulu/Bundle/ContentBundle/Tests/bootstrap.php

```diff
@@ -9,11 +9,11 @@
  * with this source code in the file LICENSE.
  */
 
-$file = __DIR__ . '/../vendor/autoload.php';
+$file = realpath(__DIR__ . '/../vendor/autoload.php');
```

Comment:
Why is this change only required in this single file, but not in all the others?